datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -47,59 +70,63 @@ $ docker run --rm -i --name ucp \
 $ gpg --decrypt /tmp/backup.tar | tar --list
 ```
 
-## Restore command
+## Restore your cluster
+
+The restore command can be used to create a new UCP cluster from a backup file.
+After the restore operation is complete, the following data will be recovered
+from the backup file:
+* Users, Teams and Permissions.
+* All UCP Configuration options available under `Admin Settings`, such as the
+DDC Subscription license, scheduling options, Content Trust and authentication
+backends.
+
+There restore operation can be performed in any of three environments:
```

Comment:
There -> The
You say "three environments" but you only list two. Also you can make it a bit more straightforward like

"There are two ways to restore a UCP cluster"